lib/src/deploy.rs

```diff
@@ -288,6 +288,14 @@ pub(crate) async fn prune_container_store(sysroot: &Storage) -> Result<()> {
     Ok(())
 }
 
+pub(crate) async fn wipe_ostree(sysroot: &Sysroot) -> Result<()> {
```

Comment:
We are absolutely not consistent about this, just an optional thing to maybe followup on later:

In theory an `async fn` shouldn't block the thread, but `write_deployments` is a totally blocking heavy filesystem operation. What we do in a few places is do heavy stuff like this inside a `tokio::task::spawn_blocking`. But again we're not *at all* consistent and it doesn't *really* matter right now because we aren't really doing many things concurrently at the moment.

But, the second we try to e.g. run as a systemd service that exposes an RPC API it would start to matter.